### ISSUE TITLE
Convert accordion to use component OR revisit accordion.js

### ISSUE THREAD

**Issue by magjarvis** [ATLAS-Illinois/Illinois-Framework-Theme#1044]
The accordion paragraph type is not using the aria-expanded=true/false attribute. This should be able to be resolved by either converting to the V3 component or revisiting the paragraph-accordion.js file. Per issue #280 it looks like the js file was causing some errors and was removed
